### PR TITLE
Disable beego color

### PR DESCRIPTION
**Pull request by ninjadq**
The color can not disable because beego's bug

I create a harbor log adapter which can use harbor's logger to write log msg

Acctually this commit can not disable color in core log but will reduce some useless infos

### CODE REVIEW

**Comment by heww** [comment]
If the requirement is only to disable the color of the log, maybe the following will work.

```go
	beego.SetLogger(logs.AdapterConsole, "{\"color\": false}")
```

**Comment by ninjadq** [comment]
@heww That method doesn't work. Because Beego's console adapter has a bug when creating a console adapter. 
And beego's log level doesn't controlled by Harbor config and it also not consistent with harbor. So this pr also resolved this.

File: src/common/utils/log/beegoadapter.go [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/astaxie/beego/logs"
+)
+
+// BeeLevel2Hbr mapping beego's log level to harbor's log lever
```
Comment (reasonerjt) [inline src/common/utils/log/beegoadapter.go]: Do we have to export this variable?

Comment (ninjadq) [reply src/common/utils/log/beegoadapter.go]: I think import logs are the most convenient way or I need using integers as the key which is not good for readability.

**Pull request closed without merging**